_layouts/event.html

```diff
@@ -8,7 +8,7 @@ <h1 class="post-title">{{ page.title | escape }}</h1>
   </header>
 
   {% if page.image %}
-    <img src="assets/events/{{page.image}}"/>
+    <img src="/assets/events/{{page.image}}"/>
```

Comment:
This will break when hosted by github. 

Please use {{ site.github.url }}/assets